web/src/components/Article/components/FullLayout/FullLayout.tsx

```diff
@@ -22,7 +23,8 @@ interface Props {
 const FullLayout = ({ article }: Props) => {
   const { width } = useWindowDimensions()
   const isMobile = width < 428
-  console.log('ismobile', isMobile)
+
+  const [english, setEnglish] = React.useState<boolean>(false)
```

Comment:
Can't you make the default value of this state depend on the availability of an english or dutch version of the post?

if dutch && english => default to dutch
if only english => default to english
if only dutch => default to dutch

Reply:
It is now not possible to have only english, the default title & body are required, whilest the specific english is optional